add info about hostname change (#3434)
Chocobozzz/PeerTube: support/doc/tools.md

### update-host.js

> **Changing the hostname is unsupported and may be a riskful operation (with risk probability increasing the higher your network of followers is).**

Reviewer: ```suggestion
**Changing the hostname is unsupported and may be a risky operation, especially if you have already federated.**
```

If you started PeerTube with a domain, and then changed it you will have
invalid torrent files and invalid URLs in your database. To fix this, you have

> to run the command below. That won't update the URL:s at your follower instances though.

Reviewer: ```suggestion
to run the command below (keep in mind your follower instances will NOT update their URLs).
```
